### DISCUSSION TITLE
get actual groups in suite event

### DISCUSSION THREAD

**Discussion by gimler** [Codeception/Codeception#6182] [Ideas]
#### What are you trying to achieve?
Get the actual running groups in extension suite.before event in my extension. to only execute the code if a specific group was set or not. 

#### What do you get instead?
`\Codeception\Event\SuiteEvent::getGroups()` return all groups found in the suite not only the groups from the tests.

### Details

* Codeception version: 2.2 branch
* Installation type: Composer
* List of installed packages (`composer show`)

**Comment by gimler** [comment]
Another idea is to disable the extension for a specific env. But there is no disable option and also not a `default` env.

So i can add a extension config option `disable` but i think this is not so sexy.

The best i think would be to get the active groups in the extension suite event.

**Comment by Naktibalda** [comment]
#5938 seems related. SuiteManager validates all group files even when group parameter is not used.
